script.js

```diff
@@ -0,0 +1,33 @@
+
+//properties added to the object: ‘input’ and ‘output’ as an array
+
+var chatbot = {
+    input: ["hello", "how are you", "how is your day", "what do you do"],
+    output: [ "hi", "greaatt!", "good", "software developer" ]
+ };
+
+//print variable
+//console.log(chatbot);
+
+//created a function called ‘reply’.
+function reply() {
+    let question = document.getElementById("input").value.toLowerCase();
+
+//conditional statement
+    if (chatbot.input[0] === question) {
```

Comment:
Can you think of checking for the input in a different way without all of these if and else if statements?  
For example, if I had 20 possible input/outputs, that would be a lot of if statements to write! 

<details>
<summary>Click here for a suggestion</summary>

- You could loop through the chatbot's input and compare that with the user's input to check if it's valid. If there is a match, display it. Otherwise display "enter a valid value."

- Or you could use the array [indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) method to see if the user's input is valid. If it is, display it. Otherwise, display "enter a valid value."
</details>